setup.cfg

```diff
@@ -13,6 +13,7 @@ classifiers =
     Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3.11
     Programming Language :: Python :: 3.12
+    Programming Language :: Python :: 3.13
```

Comment:
Should we pop the `Programming Language :: Python :: 3.7` classifier above?

Reply:
Good point and I was also wondering about it, here we simply remove wheels for 2.7 but users could still build manually for Python 3.7 and use it. We are still testing Python 3.7 with the VFX 2020 platform docker images, though it's about to be removed in the VFX 2024 PR. So all that to say that I have no objections to remove :)

Reply:
Looking at the Scientific Python SPEC0, support for Python 3.10 is about to be dropped: https://scientific-python.org/specs/spec-0000/, CY2023 is Python 3.10, and Python 3.7 is CY2021.

I think that as a whole, there should be an ASWF consensus on which minimum Python version to support, especially when some of the core packages we are relying on, e.g., Numpy have seen their support dropped. In Colour, we split the apple in two and support the previous VFX Reference Platform year as minimum. Let's discuss at TSC.

Reply:
I forgot that VFX 2021 is still using Python 3.7, for now let's keep it in there as there was no strong feeling against at yesterday TSC. We should definitively make sure we have a defined plan for older platform support so worth discussing at the next TSC when you are there :)